app/src/main/res/layout/fragment_edit_filter.xml

```diff
@@ -148,8 +149,7 @@
                     android:layout_height="wrap_content"
                     android:hint="@string/package_name"
                     app:boxCornerRadiusTopStart="10dp"
-                    app:boxCornerRadiusTopEnd="10dp"
-                    android:layout_marginEnd="20dp"
+                    app:boxCornerRadiusTopEnd="8dp"
```

Comment:
The others have `10dp`, why would this one have `8dp`?

Reply:
I'm very sorry, I was going to add a bit of margin and changed it by mistake